pkg/pillar/types/domainmgrtypes.go

```diff
@@ -204,6 +203,7 @@ type DiskStatus struct {
 	Devtype            string // From config
 	Vdev               string // Allocated
 	ActiveFileLocation string // Allocated; private copy if RW; FileLocation if RO
+	FSVolumeLocation   string // Allocated; for containers this has path to the FSVolume
```

Comment:
Couldn't we use ActiveFileLocation for this? 
That would have the benefit of the size being reported to the controller.